plugins/org.eclipse.fordiac.ide.ui/src/org/eclipse/fordiac/ide/ui/widget/NatTableWidgetFactory.java

```diff
@@ -173,7 +182,7 @@ public static NatTable createRowNatTable(final Composite parent, final DataLayer
 			public void configureUiBindings(final UiBindingRegistry uiBindingRegistry) {
 				uiBindingRegistry.unregisterKeyBinding(new KeyEventMatcher(SWT.MOD1, 'c'));
 				uiBindingRegistry.unregisterKeyBinding(new KeyEventMatcher(SWT.MOD1, 'x'));
-				uiBindingRegistry.registerKeyBinding(new KeyEventMatcher(SWT.MOD1, 'c'), new CopyDataFromTableAction());
+				uiBindingRegistry.registerKeyBinding(new KeyEventMatcher(SWT.MOD1, 'c'), copyAction);
 				uiBindingRegistry.registerKeyBinding(new KeyEventMatcher(SWT.MOD1, 'v'),
 						new PasteDataIntoTableAction(section));
```

Comment:
I would suggest to definine custom command handlers for the `CopyDataToClipboardCommand` and `PasteDataCommand` instead of overriding the default `CopyDataAction` and `PasteDataAction`.